src/v2/lib/entity-handlers.js

```diff
@@ -85,7 +90,12 @@ const getEntity = async (request, key) => {
   // e.g. contactsService.getContact(id)
   const entity = await services[key][getFn](id)
 
-  return entity || Boom.notFound(`No ${lowerCase(key)} found for ${id}`)
+  return entity || Boom.notFound(`No ${splitUppercase(key)} found for ${id}`)
+}
+
+function splitUppercase (key) {
+  const results = key.replace(/([A-Z])/g, ' $1')
```

Comment:
Again, would be great to have a brief comment that explains how this regex is working.